### PR TITLE
`Upgrade Badger v1 to v3` content redirect

### LINKED ISSUE #682: Move Badger migration instructions to another doc
This section https://www.jaegertracing.io/docs/1.55/deployment/#badger---local-storage contains a large subsection related to Badger v1->v3 migration. It does not need to be in the docs. I suggest moving it to a an .md file in the `jaeger` repository inside Badger directories and link from the docs.

### PR DESCRIPTION
**Pull request by tico88612**
## Which problem is this PR solving?
- Resolves #682
- Related https://github.com/jaegertracing/jaeger/pull/5276

## Description of the changes
- `Upgrade Badger v1 to v3` redirect to `jaeger`

## How was this change tested?
- Manual test

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits
- [ ] I have added unit tests for the new functionality
- [ ] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

File: content/docs/1.55/deployment.md [hunk 1]
```diff
@@ -279,116 +279,7 @@ docker run \
   jaegertracing/all-in-one:{{< currentVersion >}}
 ```
 
-#### Upgrade Badger v1 to v3
```
Comment (yurishkuro) [inline content/docs/1.55/deployment.md:282]: make this change in `content/docs/latest/deployment.md`, not this file

Comment (tico88612) [reply content/docs/1.55/deployment.md:282]: I didn't find the `latest/` folder, which contains the latest version. Do you mean `next-release/`?

Comment (yurishkuro) [reply content/docs/1.55/deployment.md:282]: yes

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
thanks!

**Pull request merged**